assignment/impl/src/java/org/sakaiproject/assignment/impl/GradeSheetExporter.java

```diff
@@ -233,7 +233,12 @@ public void getGradesSpreadsheet(final OutputStream out, final String reference,
 
                                                 if (grade == null) {
                                                     // TODO originally called submission.getGradeForUser(userId);
-                                                    grade = submissionSubmitter.getGrade() != null ? submissionSubmitter.getGrade() : submission.getGrade();
+                                                	if(submissionSubmitter.getGrade() != null) {
```

Comment:
There shouldn't be a difference between the assignments grade value and the submitters grade value meaning they both should have the same format.

Reply:
Can you please confirm if it is ok now?